### PR TITLE
[BugFix] fix problems found in bug bash

### PR DESCRIPTION
**Pull request by kayzliu**
## Description

1. fix inconsistent argument name (number_workers and num_workers) in README of graphsim
2. fix inappropriate line breaks in docstring of fakenews.py
3. fix deprecated graph type in fraud dataset
4. fix incorrect example in fraud dataset
5. fix docstring and example in knn_graph
6. fix incorrect node type name

@mufeili 

## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change

### CODE REVIEW

File: python/dgl/data/fakenews.py [hunk 1]
```diff
@@ -20,18 +20,10 @@ class FakeNewsDataset(DGLBuiltinDataset):
...
-
-    - profile: the 10-dimensional node feature composed of ten Twitter
-    user profile attributes.
-
-    - spacy: the 300-dimensional node feature composed of Twitter user
-    historical tweets encoded by the spaCy word2vec encoder.
+    - bert: the 768-dimensional node feature composed of Twitter user historical tweets encoded by the bert-as-service
+    - content: the 310-dimensional node feature composed of a 300-dimensional “spacy” vector plus a 10-dimensional “profile” vector
+    - profile: the 10-dimensional node feature composed of ten Twitter user profile attributes.
+    - spacy: the 300-dimensional node feature composed of Twitter user historical tweets encoded by the spaCy word2vec encoder.
```
Comment (mufeili) [inline python/dgl/data/fakenews.py:26]: Have you tried locally rendering the document?

**Comment by mufeili** [comment]
Thanks a lot! I'm good other than a minor comment.

File: python/dgl/data/fakenews.py (hunk 1 above)
Comment (kayzliu) [reply python/dgl/data/fakenews.py:26]: Actually not, but I followed the style of existing docstring. If you can provide the instruction on how to render the document locally, I will be happy to do so.

Comment (BarclayII) [reply python/dgl/data/fakenews.py:26]: You can follow the instructions in `docs/README.md`.  Note that this builds MXNet, PyTorch and Tensorflow documentations in one shot.

**Review by BarclayII — APPROVED** [review]

File: python/dgl/data/fakenews.py (hunk 1 above)
Comment (kayzliu) [reply python/dgl/data/fakenews.py:26]: Thanks a lot!

**Pull request merged**